docs/lang/articles/reference.md

```diff
@@ -250,8 +250,56 @@ with `expression2` as the error message.
 
 ## Compound statements
 
+This section describes the supported syntax of compound statements in Taichi,
+and it is modeled after [the compound statement chapter of The Python Language Reference](https://docs.python.org/3/reference/compound_stmts.html)
+
+A compound statement consists of one or more ‘clauses’.
```

Comment:
For writing consistency of this article, let's use *clauses* instead of ‘clauses’.